samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -269,7 +372,11 @@ int location_shell(const struct shell *shell, size_t argc, char **argv)
 			gnss_num_fixes = atoi(optarg);
 			gnss_num_fixes_set = true;
 			break;
-
+		case LOCATION_SHELL_OPT_GNSS_LOC_CLOUD_NMEA:
+			gnss_location_to_cloud_format = CLOUD_GNSS_FORMAT_NMEA;
```

Comment:
Add comment for flow-through, just to know it's intentional?

Reply:
Added